test/lib/utils.test.js

```diff
@@ -1,24 +1,22 @@
+jest.useFakeTimers();
+
+jest.mock("ldclient-js");
+
+import { initialize } from "ldclient-js";
+
 describe("lib/utils", () => {
-  jest.useFakeTimers();
-
-  let utils = require("./../../src/lib/utils");
-  let ldClient = require("ldclient-js").default;
-
-  const mockLdClient = (() => {
-    ldClient.identify = jest.fn();
-
-    ldClient.initialize = jest.fn().mockImplementation(() => ({
-      on: (event, callback) => {
-        setTimeout(() => {
-          callback();
-        }, 1000);
-      },
-      variation: jest.fn,
-      track: jest.fn,
-      identify: jest.fn,
-    }));
-  });
-  mockLdClient();
+  initialize.mockImplementation(() => ({
+    on: (event, callback) => {
+      setTimeout(() => {
+        callback();
+      }, 1000);
+    },
+    variation: jest.fn,
+    track: jest.fn,
+    identify: jest.fn,
+  }));
+
+  const utils = require("./../../src/lib/utils");
```

Comment:
Because we're no longer doing the manual node module shenanigans, we can revert this to an `import`.